modules/aggregation-layers/src/heatmap-layer/triangle-layer.js

```diff
@@ -65,7 +65,11 @@ export default class TriangleLayer extends Layer {
   draw({uniforms}) {
     const {model} = this.state;
     const {texture, maxTexture, colorTexture, intensity, threshold} = this.props;
-    model.setUniforms({texture, maxTexture, colorTexture, intensity, threshold}).draw();
+    model
```

Comment:
I know you prefer Object.assign, which is fine. Since ES2018 now finally supports spread operator `...` as part of the JS standard, other repos like loaders.gl are adopting it.